R/proportional_hazards.R

```diff
@@ -65,9 +65,27 @@ fit.proportional_hazards <- function(object,
   # call parsnip::fit.model_spec()
   res <- NextMethod()
 
+  # TODO find a public place to document that
```

Comment:
See my note below about documenting the strata stuff for glmnet in `glmnet_fit_wrapper()`